client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -464,3 +475,15 @@ export const getSiteCountText = ( sites: Array< Site > ) => {
 		comment: '%(siteCount) is no of sites selected, e.g. "2 sites"',
 	} );
 };
+
+export const getHumanReadableCount = ( count: number ) => {
+	if ( count > 1000 ) {
+		return translate( '%(count)dk', {
+			args: {
+				count: Math.round( count / 1000 ),
+			},
+			comment: 'Show the count in thousands',
+		} );
+	}
+	return count;
+};
```

Comment:
In the main column, I used `ShortenedNumber from @automattic/components` not sure if this makes a difference or not. Is this way better? Or use the component?

I think we should be consistent and pick one way to do it so the numbers will always be consistent.

Reply:
Thanks for the review, guys. I have made the changes accordingly - https://github.com/Automattic/wp-calypso/pull/73730/commits/e6ba55b13ad3be95f1f9f5fd4d71fcf0b58e6e82